tests/insertion.rs

```diff
@@ -404,6 +404,102 @@ fn after_removal() {
         .single(client_app.world());
 }
 
+#[test]
+fn dont_replicate_after_connect() {
+    let mut server_app = App::new();
+    let mut client_app = App::new();
+    for app in [&mut server_app, &mut client_app] {
+        app.add_plugins((
+            MinimalPlugins,
+            RepliconPlugins.set(ServerPlugin {
+                tick_policy: TickPolicy::EveryFrame,
+                replicate_after_connect: false,
+                ..Default::default()
+            }),
+        ))
+        .replicate::<TableComponent>()
+        .replicate::<SparseSetComponent>();
+    }
+
+    server_app.connect_client(&mut client_app);
+
+    let client_id = client_app
+        .world()
+        .resource::<RepliconClient>()
+        .id()
+        .unwrap();
+
+    server_app.world_mut().spawn((Replicated, TableComponent));
+
+    server_app.update();
+    server_app.exchange_with_client(&mut client_app);
+    client_app.update();
+    server_app.exchange_with_client(&mut client_app);
+
+    // Test that we're not replicating to this client yet.
+    assert_eq!(
+        client_id,
+        server_app
+            .world()
+            .resource::<ConnectedClients>()
+            .iter_client_ids()
+            .next()
+            .unwrap()
```

Comment:
Could you create a separate variable for resources in assertions like this to make it format a little nicer?

Reply:
fixed